va/va_enc_av1.h

```diff
@@ -0,0 +1,974 @@
+            uint32_t    enable_warped_motion                        : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_dual_filter                          : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_order_hint                           : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_jnt_comp                             : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_ref_frame_mvs                        : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_superres                             : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_cdef                                 : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_restoration                          : 1;
+            /** \brief Reserved bytes for future use, must be zero. */
+            uint32_t    reserved_bits                               : 18;
+        } bits;
+        uint32_t value;
+    } seq_fields;
+
+    /** \brief Corresponds to AV1 syntax element of the same name.
+     *  value range [0..7].
+     */
+    uint8_t     order_hint_bits_minus_1;
+
+    /** \brief Reserved bytes for future use, must be zero */
+    uint32_t                va_reserved[VA_PADDING_HIGH];
+} VAEncSequenceParameterBufferAV1;
+
+#define VA_AV1_MAXNUM_SEGMENTS                  8
+#define VA_AV1_MAXNUM_SEGFEATURES               8
+
+/**
+ * \brief Segment parameters
+ */
+typedef struct _VAEncSegParamAV1 {
+    union {
+        struct {
+            /** \brief Indicates if segmentation is enabled in the current frame.
+             *  If disabled, all the below parameters in the structure should
+             *  be set to 0, and ignored by driver.
+             */
+            uint8_t     segmentation_enabled            : 1;
+
+            /** \brief when segmentation_enabled equals 1 and segment_number > 0,
+             *  this parameter equals 1 indicates the segmentation map may
+             *  come from DDI, and that the Segmentation Map Data Buffer
+             *  should be provided with populated segment_id. If equals 0,
+             *  segmentation map should be inherited from a reference frame
+             *  (specified by primary_ref_frame). When segmentation_enabled or
+             *  SegmentNumber equals 0, this parameter should be set to 0
+             *  and ignored by driver.
+             */
+            uint8_t     segmentation_update_map         : 1;
+            uint8_t     segmentation_temporal_update    : 1;
+            /** \brief Reserved bytes for future use, must be zero. */
```

Comment:
missing segmentation_update_data?

Reply:
update data  will be transfered  through buffer VAEncPictureParameterBufferAV1.